packages/react/src/Prose/Prose.tsx

```diff
@@ -0,0 +1,29 @@
+import React, {forwardRef} from 'react'
+import clsx from 'clsx'
+import type {BaseProps} from '../component-helpers'
+import styles from './Prose.module.css'
+import '@primer/brand-primitives/lib/design-tokens/css/tokens/functional/components/prose/base.css'
+
+export type ExpectedProseChildren = HTMLDivElement | HTMLHeadingElement | HTMLParagraphElement
```

Comment:
Is this for type-narrowing? I can't see where it's being used.

Reply:
This has been deleted as it was old code in the following: [SHA](https://github.com/primer/brand/pull/316/commits/9a113fa56cfe60cf197437f3f708778c95eb8aab)